### PR TITLE
makes holo damage messages more subtle if the damage is lower than 10

### PR DESCRIPTION
**Pull request by ShadowDeath6**
title

if there's a better way to show adjacent people messages please tell me

:cl: ShadowDeath6
rscadd: Holoparasites have more subtle damage messages when their damage taken is lower.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
> removed merge ready 

steaming like broccoli

**Comment by Cruix** [comment]
Sorry

Add a new parameter to visible_message for range and use that, so this respects things like blindness and invisibility

**Comment by ShadowDeath6** [comment]
I have no idea how to do that honestly

**Comment by Cruix** [comment]
I made a PR to you branch.

**Comment by ShadowDeath6** [comment]
Why god

**Comment by ShadowDeath6** [comment]
that took longer than I'm willing to admit

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -88,7 +88,7 @@ var/next_mob_id = 0
 	var/turf/T = get_turf(src)
```
Comment (Cruix) [inline code/modules/mob/mob.dm:88]: ^ You missed a spot

Comment (Cruix) [inline code/modules/mob/mob.dm:88]: ^ You missed a spot (just barely outside the diff!)

Comment (ShadowDeath6) [reply code/modules/mob/mob.dm:88]: https://github.com/ShadowDeath6/-tg-station-yog-rebase/pull/1/commits/79e4517435e4857bbd35f48da89db39d37bb94bc

uh

Comment (ShadowDeath6) [reply code/modules/mob/mob.dm:88]: oh I just realized what you meant, fuck

at least the compiler helped me fix it before you had to

**Pull request merged**